### DISCUSSION TITLE
Custom prompt for lens for change language

### DISCUSSION THREAD

**Discussion by ryush00** [GitbookIO/community#263] [Feature Requests]
I want to change the display language of the results in Lens to the Korean language that my customers are using, but the results are only displayed in English.

I would like to have the ability to change the language or set my own prompt.

**Comment by addisonschultz** [comment]
Hi @ryush00 - thanks for the feedback! I also agree that seeing the prompt in a more contextual language would be a great feature addition. I've added it for us to look into!
